src/Utils/resizer.ts

```diff
@@ -1,14 +1,20 @@
 import qs from "qs"
 const GEMINI_CLOUDFRONT_URL = "https://d7hftxdivxxvm.cloudfront.net"
+const GEMINI_DISPLAY_CLOUDFRONT_URL = "https://de5y2r7wr8mpb.cloudfront.net"
```

Comment:
I don't think it matters match currently, but these _might_ be better as ENV/config values, which clients should supply.

But, this is definitely easier for now, although these can definitely change/break in the future so it'd be nice be able to update apps easily in the future.